Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apk --update --no-cache add \
   libxslt-dev \
   libxrender \
   libgomp \
+  linux-headers\
```

Comment:
As seen in https://github.com/php/php-src/issues/8681 the package is required now

Reply:
👍